src/components/HeroIcon/HeroIcon.styles.tsx

```diff
@@ -31,6 +33,8 @@ export const iconMap = {
 };
 export type IconType = keyof typeof iconMap;
 
+export const triangleFill = 'su-fill-current su-stroke-0';
```

Comment:
The triangle icon is just me filling in the chevron and removing the stroke, and stretching them out a bit 😄